v5/core/cr_authenticator.go

```diff
@@ -405,6 +406,7 @@ func (authenticator *ComputeResourceAuthenticator) RequestToken() (*IamTokenServ
 	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
 		buff := new(bytes.Buffer)
 		_, _ = buff.ReadFrom(resp.Body)
+		/* #nosec G104 */
```

Comment:
Unsure about whether this needs to be fixed instead of ignored.  This is a Low vulnerability, G104 indicates `"Errors unhandled."`